packages/models/src/lib/plugin-config-runner.ts

```diff
@@ -1,5 +1,13 @@
 import { z } from 'zod';
 import { filePathSchema } from './implementation/schemas';
+import { auditOutputsSchema } from './plugin-process-output';
+
+export const outputTransformSchema = z
+  .function()
+  .args(z.array(z.record(z.string(), z.unknown())))
```

Comment:
This is too restrictive, I believe. It could be any JSON, not necessarily an array of objects. E.g. many tools don't output the array at top-level, but it's nested within an object. Or it could even be a plain string that the transform function is gonna parse. 

```suggestion
  .args(z.unknown())
```